python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -325,6 +353,11 @@ class Meta:
         graphene.NonNull("dagster_graphql.schema.instigation.GrapheneInstigationEventConnection"),
         cursor=graphene.String(),
     )
+    jobName = graphene.String()  # for RunsFeedEntry interface - dupe of partitionSetName
```

Comment:
hmm not sure about this part of the interface

Reply:
yeah i dont know what to do here either. i was thinking about naming it something more generic like `target` but then it seems like it should encompass `assetSelection` and `assetCheckSelection` too. maybe that would be ok?

Reply:
in the end the question is how would this field be used on the client - so best path is to get @bengotow 's input either in this initial PR or maybe defer it to a second iteration